gfortran_utils.sh

```diff
@@ -97,6 +97,21 @@ if [ "$(uname)" == "Darwin" ]; then
         hdiutil attach -mountpoint /Volumes/gfortran $GFORTRAN_DMG
         sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
         check_gfortran
+        if [[ "${PLAT:-}" == "universal2" || "${PLAT:-}" == "arm64" ]]; then
+            curl -L -O https://github.com/isuruf/gcc/releases/download/gcc-10-arm-20210112/gfortran-darwin-arm64.tar.gz
+            sudo mkdir -p /opt/
+            sudo cp gfortran-darwin-arm64.tar.gz /opt/gfortran-darwin-arm64.tar.gz
+            pushd /opt
+                sudo tar -xvf gfortran-darwin-arm64.tar.gz
+                sudo rm gfortran-darwin-arm64.tar.gz
+            popd
+            export FC_ARM64="/opt/gfortran-darwin-arm64/bin/arm64-apple-darwin20.0.0-gfortran"
```

Comment:
Any way of getting this path more automatically?  Or generalizing so we don't have to change it with other releases?

Reply:
Since the archive is checked into the repo, this won't change.

Reply:
Well - sorry - I'm not saying it's necessarily a good idea to check it into the repo - if there is some other way to guarantee that we can get exactly that binary from something similar to version control elsewhere, that would be just as good.

Reply:
Reverted back to the URL and added a shasum check.